pkg/services/secrets/kvstore/plugin_fatal_crash_test.go

```diff
@@ -19,15 +22,17 @@ import (
 // Set fatal flag to true, then simulate a plugin start failure
 // Should result in an error from the secret store provider
 func TestFatalPluginErr_PluginFailsToStartWithFatalFlagSet(t *testing.T) {
-	svc, _, _, err := setupFatalCrashTest(t, true, true, false)
+	svc, mgr, _, _, err := setupFatalCrashTest(t, true, true, false)
```

Comment:
Shouldn't `mgr` be `_` if it's not being used here?

Reply:
linter wouldn't let me :( `_, _, _` is called a "dogsled" and apparently it's bad. Since it's a unit test and not actual code, I figured it was fine to just do this.